src/Sylius/Bundle/ResourceBundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -60,7 +62,32 @@ public function load(array $config, ContainerBuilder $container)
             $container->setParameter('sylius.translation.available_locales', $config['translation']['available_locales']);
         }
 
-        foreach ($config['resources'] as $alias => $resourceConfig) {
+        $container->setParameter('sylius.resource.settings', $config['settings']);
+        $container->setAlias('sylius.resource_controller.authorization_checker', $config['authorization_checker']);
+
+        $this->loadPersistence($config['drivers'], $config['resources'], $loader);
+        $this->loadResources($config['resources'], $container);
+    }
+
+    private function loadPersistence(array $enabledDrivers, array $resources, LoaderInterface $loader)
+    {
+        foreach ($resources as $alias => $resource) {
+            if (!in_array($resource['driver'], $enabledDrivers)) {
+                throw new InvalidArgumentException(sprintf(
+                    'Resource "%s" uses driver "%s", but this driver has not been enabled.',
```

Comment:
This is a sanity check. If resources have been configured and their drivers have not been enabeld, throw an exception. We magically add any missing drivers, but I prefer explicitness.